deltacat/aws/s3u.py

```diff
@@ -385,7 +397,7 @@ def download_manifest_entry(
     retrying = Retrying(
         wait=wait_random_exponential(multiplier=1, max=60),
         stop=stop_after_delay(30 * 60),
-        retry=retry_if_not_exception_type(NonRetryableError),
+        retry=retry_if_not_exception_type(DownloadTableError),
```

Comment:
Did you not make it `retry_if_exception_type(RetryableError)` earlier? We talked about finding all known errors in first couple of days and adding them to our retryable error codes.